<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Radio</class>
 <widget class="QWidget" name="Radio">
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="radioBrowserSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>My radio stations</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="myRadioListWidget">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="loadMyRadioStationButton">
           <property name="toolTip">
            <string>Load radio station list from file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="saveMyRadioStationButton">
           <property name="toolTip">
            <string>Save radio station list to file</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="addMyRadioStationButton">
           <property name="toolTip">
            <string>Add new radio station</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="editMyRadioStationButton">
           <property name="toolTip">
            <string>Edit selected radio station</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeMyRadioStationButton">
           <property name="toolTip">
            <string>Remove selected radio station</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QComboBox" name="searchByComboBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="searchComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addRadioBrowserStationButton">
           <property name="toolTip">
            <string>Add new radio station</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="radioView">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragOnly</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="LineEdit" name="filterEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>LineEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>myRadioListWidget</tabstop>
  <tabstop>searchByComboBox</tabstop>
  <tabstop>searchComboBox</tabstop>
  <tabstop>addRadioBrowserStationButton</tabstop>
  <tabstop>radioView</tabstop>
  <tabstop>addMyRadioStationButton</tabstop>
  <tabstop>editMyRadioStationButton</tabstop>
  <tabstop>removeMyRadioStationButton</tabstop>
  <tabstop>filterEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
